pkg/apis/core/types.go

```diff
@@ -2242,6 +2242,14 @@ const (
 	RestartPolicyNever     RestartPolicy = "Never"
 )
 
+const (
+	// When RestartPolicy is OnFailure, use the annotation MaxRetriesAnnotation to specify
+	// max retry times. Default value is MaxRetriesDefault, and negative value means always
+	// retry on failure.
+	MaxRetriesAnnotation = "kubernetes.io/maxRetriesForOnFailurePolicy"
```

Comment:
comments should start with the name of const of functions.

Reply:
Addressed.